src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentPrice = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days <= 2) {
+    return carRentPrice * days;
+  } else if (days <= 6) {
+    return carRentPrice * days - minDiscount;
+  }
```

Comment:
```suggestion
  if (days <= 2) {
    return carRentPrice * days;
  }
  
  if (days <= 6) {
    return carRentPrice * days - minDiscount;
  }
```
краще уникати else if, return і так завершить функцію при виконанні умови